src/verify_cert.rs

```diff
@@ -495,10 +495,9 @@ enum Role {
     EndEntity,
 }
 
-#[cfg(test)]
+#[cfg(all(test, feature = "alloc", feature = "ring"))]
```

Comment:
Thanks :+1: I also moved to a module-wide requirement for `alloc` in https://github.com/rustls/webpki/pull/179